packages/theming/default-theme/src/defaultTheme.ts

```diff
@@ -88,6 +88,65 @@ function _defaultTypography(): Typography {
   return defaultsDict;
 }
 
+export function defaultEffects(): Effects {
+  const effects =
+    Platform.OS === 'android' || Platform.OS === 'ios'
+      ? {
+          borderRadiusSmall: 2,
+          borderRadiusMedium: 4,
+          borderRadiusLarge: 8,
+          borderRadiusXLarge: 12,
+        }
+      : {
+          borderRadiusSmall: 2,
+          borderRadiusMedium: 4,
+          borderRadiusLarge: 6,
+          borderRadiusXLarge: 8,
+        };
+
+  const noOffset = { width: 0, height: 0 };
+  const nearShadowAmbient = { shadowOffset: noOffset, shadowOpacity: 0.12, shadowRadius: Platform.OS === 'ios' ? 1 : 2 };
+  const farShadowAmbient = { shadowOffset: noOffset, shadowOpacity: 0.2, shadowRadius: Platform.OS === 'ios' ? 4 : 8 };
+  Object.assign(
+    effects,
+    ((Platform.OS === 'ios' || Platform.OS === 'windows' || Platform.OS === ('win32' as any) || Platform.OS === 'web') && {
+      shadow2: {
+        ambient: nearShadowAmbient,
+        key: { shadowOffset: { width: 0, height: 1 }, shadowOpacity: 0.14, shadowRadius: Platform.OS === 'ios' ? 1 : 2 },
+      },
+      shadow4: {
+        ambient: nearShadowAmbient,
+        key: { shadowOffset: { width: 0, height: 2 }, shadowOpacity: 0.14, shadowRadius: Platform.OS === 'ios' ? 2 : 4 },
+      },
+      shadow8: {
+        ambient: nearShadowAmbient,
+        key: { shadowOffset: { width: 0, height: 4 }, shadowOpacity: 0.14, shadowRadius: Platform.OS === 'ios' ? 4 : 8 },
+      },
+      shadow16: {
+        ambient: nearShadowAmbient,
+        key: { shadowOffset: { width: 0, height: 8 }, shadowOpacity: 0.14, shadowRadius: Platform.OS === 'ios' ? 8 : 16 },
+      },
+      shadow28: {
+        ambient: farShadowAmbient,
+        key: { shadowOffset: { width: 0, height: 14 }, shadowOpacity: 0.24, shadowRadius: Platform.OS === 'ios' ? 14 : 28 },
+      },
+      shadow64: {
+        ambient: farShadowAmbient,
+        key: { shadowOffset: { width: 0, height: 32 }, shadowOpacity: 0.24, shadowRadius: Platform.OS === 'ios' ? 32 : 64 },
+      },
+    }) ||
+      ((Platform.OS === 'android' || Platform.OS === 'macos') && {
+        shadow2: undefined,
+        shadow4: undefined,
+        shadow8: undefined,
+        shadow16: undefined,
+        shadow28: undefined,
+        shadow64: undefined,
+      }),
+  );
+  return effects as Effects;
```

Comment:
A lot of these checks might be easier if we assume that `default` means web/win32, and have the default theme (if no theme is specified) be the platform themes (android/apple theme). That way, we could move the platform specifics to the platform themes and feel less bad lol. How would you feel about that?